examples/vercel-ai-chatbot/app/api/chat/route.ts

```diff
@@ -92,19 +92,12 @@ export async function POST(req: Request) {
             const stringOutputType = event.data.outputs?.find(
               (o): o is WorkflowOutput.String => o.type === 'STRING'
             )
-            await kv.hmset(`chat:${id}`, {
-              id,
-              createdAt: Date.now(),
-              userId,
+            await kv.hset(`chat:${id}`, {
               messages: messages.concat({
                 role: 'ASSISTANT' as const,
                 content: stringOutputType
               })
             })
-            await kv.zadd(`user:chat:${userId}`, {
```

Comment:
we were adding an entry to redis on every message? comeon Vercel

Reply:
Is this to set `score` for chat ordering?

Reply:
I suppose to keep the last edited at the top?